### DISCUSSION TITLE
Winget shows CLSID instead of package ID and source as none even though I installed the package from winget

### DISCUSSION THREAD

**Discussion by hwfanatic** [microsoft/winget-cli#3812] [Q&A] [answered]
I've installed "Microsoft Visual C++ 2005 Redistributable (x64)" through winget, but winget list shows this package ID as "{ad8a2fa1-06e7-4b0d-927d-6e54b3d31028}" instead of Microsoft.VCRedist.2005.x64 and source as empty instead of winget.

What could be the cause of this issue and how would I go about fixing it? So far, I've tried uninstalling and reinstalling the package without success.

For example "Microsoft Visual C++ 2005 Redistributable" doesn't have this problem and shows the correct package ID Microsoft.VCRedist.2005.x86 and source as winget.

**Comment by Trenly** [comment, marked as answer]
There are several known issues with matching, and right now there isn't anything you can do about it. VCRedists are specifically known to have this issue, as are many other packages. These requests are all being tracked using the https://github.com/microsoft/winget-cli/labels/Area-Matching label.

**Reply by hwfanatic** [reply]
I presume I don't have to create a new issue out of this. Do I?

**Reply by Trenly** [reply]
I don't think so; I'll try and find the ones on vcredist that I can link back here
